examples/mecci_kissinger.py

```
@@ -0,0 +1,234 @@
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from model_fitting_methods.kissinger import kissinger_method
+from data_import.dsc_importer import dsc_importer
+from result_visualization.kinetic_plot import plot_kissinger
+from scipy.signal import savgol_filter
```

Ruff

`scipy.signal.savgol_filter` imported but unused

Remove unused import: `scipy.signal.savgol_filter`

(F401)